internal/postprocessor/main.go

```diff
@@ -280,23 +341,24 @@ func docURL(cloudDir, importPath string) (string, error) {
 	return "https://cloud.google.com/go/docs/reference/" + mod + "/latest/" + pkgPath, nil
 }
 
-func (c *config) AmendPRDescription(ctx context.Context) error {
+func (c *config) AmendPRDescription(ctx context.Context) ([]string, error) {
```

Comment:
Can you add docs to this function. Without them it is unclear why or what this function is returning by reading the signature.

Reply:
I think I fixed this by changing the function to return only an error.